pkgs/top-level/python-packages.nix

```diff
@@ -21394,11 +21394,13 @@ in {
   };
 
   sqlmap = buildPythonPackage {
```

Comment:
@Mic92 moved it out of pythonPackages.
Is the callPackage (https://github.com/NixOS/nixpkgs/pull/28314/files#diff-2afbba83e1b14f9e0c304c769831a0e2R21396) in pythonPackages ok or should I remove that?